tests/test_badpixel.py

```diff
@@ -0,0 +1,91 @@
+import os
+import pytest
+import corgidrp
+import numpy as np
+import corgidrp.mocks as mocks
+from corgidrp.mocks import create_default_headers
+from corgidrp.l2a_to_l2b import correct_bad_pixels
+from corgidrp.data import Image, Dataset, BadPixelMap
+
+old_err_tracking = corgidrp.track_individual_errors
+
+data = np.ones([1024,1024])*2.
+err = np.ones([1024,1024]) *0.5
+dq = np.zeros([1024,1024], dtype = np.uint16)
+prhd, exthd = create_default_headers()
+
+def test_bad_pixels():
+
+    corgidrp.track_individual_errors = True
+
+    print("UT for pipeline step correct_bad_pixels")
+
+    image1 = Image(data, pri_hdr = prhd, ext_hdr = exthd, err = err, dq = dq)
+    dataset = Dataset([image1])
+
+    corgidrp.track_individual_errors = old_err_tracking
+
+    data_cube = dataset.all_data # 3D data cube for all frames in the dataset
+    dq_cube = dataset.all_dq # 3D DQ array cube for all frames in the dataset
+    # Add some CR
+    col_cr_test = [12, 123, 234, 456, 678, 890]
+    row_cr_test = [546, 789, 123, 43, 547, 675]
+    for i_col in col_cr_test:
+        for i_row in row_cr_test:
+            dq_cube[0, i_col, i_row] += 128
```

Comment:
Similar to above, can we do this by setting the bit?

Reply:
Done

Reply:
UT passed